OKTA-413125 post tuscany edits part3 and nut-fact additions (#2852)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/nodejs/integrationsteps.md

### 2: Add JavaScript to initialize and load the Widget

> Load the Widget in the sign-in page, similar to the following snippet:

Reviewer: ```suggestion
Load the Widget on the sign-in page, similar to the following snippet:
```

</div>

>**Note:** The **Forgot password?** and **Sign Up** links are configurable elements in your

> Okta org and may not show in the screen.

Reviewer: ```suggestion
Okta org and may not show on the page.
```
